src/CSAccounting.sol

```diff
@@ -612,8 +669,18 @@ contract CSAccounting is
         uint256 nonWithdrawnKeys = MODULE.getNodeOperatorNonWithdrawnKeys(
             nodeOperatorId
         );
-        unchecked {
+        {
             uint256 currentBond = CSBondCore.getBond(nodeOperatorId);
+            {
+                uint256 reservedBond = BondReserve.getReservedBond(
+                    nodeOperatorId
+                );
+                // NOTE: `reservedBond` cannot exceed `currentBond` because it can be created
+                // only from the excess bond portion (current - required > 0).
```

Comment:
```suggestion
                // only from the excess bond portion (current - required >= 0).
```

Reply:
But it wouldn't make sense because you can't create reserve with `0`